src/app/components/SummaryHighlightItem/SummaryHighlightItem.tsx

```diff
@@ -0,0 +1,31 @@
+import classes from './SummaryHighlightItem.module.css';
+
+// timestamp in seconds
+function createTimeString(timestamp: number): string {
```

Comment:
Instead of the comment, you could name the function like it (functions === comments)

```
function createTimestampInSeconds(timestamp: number): string {
```

But I don't think this is what happens here. It's not in seconds only, it displays minutes and seconds.